examples/prisma.md

```diff
@@ -0,0 +1,118 @@
+# Using Prisma on Cloudflare Workers
+
+There are two ways to use Prisma with Cloudflare Workers, we will be using Prisma Accelerate, but you can also use the Prisma [Driver Adapter](https://www.prisma.io/docs/orm/overview/databases/database-drivers)
+
+### Install Prisma
+
+Install Prisma on your Hono Cloudflare Workers. Here, I am using neon.tech free tier as my PostgreSQL database, but you can use whichever database suits your project.
+
+Go to [neon.tech](https://neon.tech/) and create a free PostgreSQL database.
+
+```ts
+npm i prisma --save-dev
+npx prisma init
+```
+
+## Setup Prisma Accelerate
+
+To setup Accelerate, go to [Prisma Accelerate](https://www.prisma.io/data-platform/accelerate?via=start&gad_source=1&gclid=CjwKCAjwvIWzBhAlEiwAHHWgvX8l8e7xQtqurVYanQ6LmbNheNvCB-4FL0G6BFEfPrUdGyH3qSllqxoCXDoQAvD_BwE) and log in.
+
+After logging in, you will be taken to a page where you can create a new Accelerate project. and log in or register for free
+
+![Accelerate Page](/images/prismaAcceleratePage.png)
+
+Create a new project by clicking the `New project` button, and name your project.
+
+![Accelerate Page](/images/accelerateCreateProject.png)
+
+You will then be taken to a page like the one below:
+
+![Accelerate Edit Page](/images/accelerateProjectPage.png)
+
+Click the `Enable Accelerate` button, and you will be taken to the following page:
+
+![Enable Page](/images/EnableAccelerate.png)
+
+Paste your neon.tech database connection string into the `database connection string` field, choose your region, and click the `Enable Accelerate` button.
+
+You will see something like this:
+
+![API Key](/images/generateApiKey.png)
+
+Click `Generate API Key` and you will receive a new API key similar to the one below:
+
+```ts
+DATABASE_URL="prisma://accelerate...."
+```
+
+Copy this `DATABASE_URL` and store it in `.dev.vars` and `.env` so that prisma cli can access it later on.
+
+
+## Set Up Prisma in Your Project
+The neon.tech URL you received can also be used as an alternative and provide Prisma with more options, so store it for later use:
+
+
+::: code-group  
+```ts [.dev.vars]  
+DATABASE_URL="your_prisma_accelerate_url"  
+DIRECT_URL="your_neon_tech_url"  
+``` 
+:::
+
+
+Now, go to your `schema.prisma` file and set the URLs like this:
+
+::: code-group
+```ts [.env]
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+}
+```
+:::
+
+Create a function like this, which you can use in your project later:
+::: code-group
+```ts
```

Comment:
What about adding the file name?

Reply:
@yusukebe, bro, I have changed all the necessary things that you asked, but I did not mention the file name because we can make this function anywhere we want, so it's not file-specific. You can make this in your root folder and use it anywhere you want, so I am not sure what file name to put here. If you want me to write that in detail, let me know.

Reply:
So the main file you can call `index.ts` and this file you can name it whatever the import is on the index.ts file. That way it's clear to people that two separate files are being working in

Reply:
Currently, the tab names are `ts`:

![CleanShot 2024-06-21 at 15 05 55@2x](https://github.com/honojs/website/assets/10682/d79566a5-6e3c-454a-b7b9-489462901e1f)

How about adding a file name like this?

```
::: code-group
```ts [filename.ts]
```

Reply:
@yusukebe 
bro, can i name the main working file as `main.ts` and the function file as `index.ts`,
or the main working file as `index.ts` and the function file as `prismafunction.ts`